src/Lykke.Job.BlockchainCashinDetector/Workflow/PeriodicalHandlers/DepositWalletsBalanceProcessingPeriodicalHandler.cs

```diff
@@ -86,7 +89,7 @@ private async Task ProcessBalancesAsync(ITimerTrigger timer, TimerTriggeredHandl
 
             var balanceProcessor = new BalanceProcessor(
```

Comment:
Can not we cache BalancesProcessor somehow?

Reply:
We don't need this since BalanceProcessor is light weight object.

Reply:
+